Tasks/AzureTestPlanV0/Invokers/jestinvoker.ts

```diff
@@ -1,14 +1,39 @@
 import tl = require('azure-pipelines-task-lib/task');
 import utils = require('../utils');
 import constants = require('../constants');
+import tr = require("azure-pipelines-task-lib/toolrunner");
+import { executeJestCommand } from '../testLibExecutor';
 
+//Jest command like >set JEST_JUNIT_OUTPUT_NAME=TEST-Jest0-junit.xml
+//>npx jest --ci --reporters=default --reporters=jest-junit -t "JestTestName"
 export async function executeJestTests(testsToBeExecuted: string[]): Promise<number> {
 
-    // jest execution will be added
-    /*executable = npm
-    args = test
-    */
+    let finalStatus = 0; 
+    let jestNpm = tl.which("npm", true);
+    await executeJestCommand(jestNpm, constants.INSTALL_JESTJUNIT);
 
-    console.log("jest changes1");
-    return 1;
+    //testsToBeExecuted: DescribeD TestH.DescribeD TestH,DescribeD TestG.DescribeD TestG,DescribeC TestF.DescribeC TestF,DescribeC TestE.DescribeC TestE,DescribeB TestD.DescribeB TestD,DescribeB TestC.DescribeB TestC,DescribeA TestB.DescribeA TestB,DescribeA TestA.DescribeA TestA
```

Comment:
trim this or add better sample for testsToBeExecuted

Reply:
github link